File: ERFINV/erfinv.py
from time import time
from scipy.special import erfinv

print(erfinv(0.5))
print(erfinv(0.9))
print(erfinv(0.99))

start = time()

N = int(1E6)
for i in range(N):
  erfinv(i/N)

print(str(time()-start) + " sec")

File: ERFINV/erfinv.jl
using SpecialFunctions

println(erfinv(0.5))
println(erfinv(0.9))
println(erfinv(0.99))

N = 1E8

function main()
  for i in 1:N
    erfinv((i-1)/N)
  end
end

@time main()
